src/components/Explore/ExplorePost.jsx

```diff
@@ -0,0 +1,90 @@
+import { useAuth, usePosts } from 'contexts'
+import { getDate, getTime } from 'utils'
+import styles from './explore.module.css'
+
+const ExplorePost = ({ post: { _id, username, content, likes: { likeCount }, createdAt } }) => {
+    const { isUserLoggedIn } = useAuth()
+    const { likePost, dislikePost, bookmarkPost, postsDispatch } = usePosts()
+
+    const handlePostBookmark = async () => {
+        if (isUserLoggedIn) {
+            const response = await bookmarkPost(_id)
+            console.log(response)
```

Comment:
can remove console if not needed further

Reply:
Thanks for pointing it out.